.gitignore

```diff
@@ -8,5 +8,7 @@ bin/
 commands.log
 .vscode/launch.json
 .vscode/settings.json
+.pytest_cache/
```

Comment:
integration mistake, this is now in here twice.

Reply:
One copy removed